test/validator_test.exs

```diff
@@ -26,6 +26,18 @@ defmodule ValidatorTest do
       assert Validator.validate(board, user_input) == {:invalid, :out_of_bounds}
     end
 
+    test "returns an error when the user enters an input where the space is taken" do
```

Comment:
Should it return `invalid` instead of `error`?

Reply:
That is a good point and something I didn't notice at first, based on the current assertion and the context of the other modules, I think this test description should say `"returns an invalid paired with the reason..."`. What are your thoughts? Thank you for catching this 😄!

Reply:
I went with `"returns an invalid error paired with the reason..."` for now, if this you think it would be better as one of the more concise ones I can always change it 😄

Reply:
Yea sounds good to me.